sentinel/tests/unit/replications.js

```diff
@@ -0,0 +1,50 @@
+var _ = require('underscore'),
+    later = require('later'),
+    sinon = require('sinon'),
+    assert = require('chai').assert,
+    config = require('../../src/config'),
+    replications = require('../../src/schedule/replications');
+
+describe('replications', () => {
+  afterEach(() => sinon.restore());
+
+  beforeEach(() => { process.env.TEST_ENV = true; });
```

Comment:
It's a bit risky to mess around with environment variables in a test, particularly if you're not tearing them down afterwards.

Also, I don't know what this one does, usually we use `UNIT_TEST_ENV`. So I'm guessing you can just remove it?

Reply:
Removed